content/account_management/index.md

```diff
@@ -0,0 +1,83 @@
+---
+title: Account Management 
+kind: documentation
+autotocdepth: 2
+customnav: accountmanagementnav
+alias: 
+    - /guides/billing
+---
```

Comment:
@l0k0ms Please discuss with @marcweisman . He  has requested pricing not be present in the docs site. It should remain in the KB or link to the pricing page.

Reply:
dully noted.